modules/ROOT/pages/dataweave-formats-excel.adoc

```diff
@@ -252,6 +254,8 @@ Valid values are `true` or `false`.
 |`ignoreEmptyLine`  |`Boolean`|`true`|Ignores an empty line by default.
 
 Valid values are `true` or `false`.
+|`maxEntrySize`  |`Number`|`null`|Sets the maximum amount of bytes a single entry in a zip-file should become.
```

Comment:
suggesting "Sets the maximum number of bytes a single entry in a ZIP file can have." or something like that in the source